### ISSUE TITLE
Shadowsocks OTA deprecated - what does this mean for protection from GFW machine learning for China users?

### ISSUE THREAD

**Issue by ortonomy** [shadowsocks/ShadowsocksX-NG#227]
Hey, anybody know about this: http://shadowsocks.org/en/spec/one-time-auth.html 

The page reads:

```
One Time Auth
**(Deprecated)** (my emphasis)
One-time authentication (shortened as OTA) is a new experimental feature designed to improve the security against CCA. You should understand the protocol before reading this document.
```

Is there a replacement technology planned? Does anyone know the timeline for removal? Support for Shadowsocks is now in a Google group which I do not have access to. Thoughts? Feelings?

**Comment by ortonomy** [comment]
When will Shadowsocks-NG implement AEAD?

**Comment by qiuyuzhou** [comment]
Shadowsocks-NG is only an ui front end of shadowsocks-libev.
After shadowsocks-libev implement AEAD, I will upgrade the ss-local binary embed in Shadowsocks-NG bundle.

**Comment by darkshell** [comment]
Shadowsocks-libev has implemented AEAD since release 3.0.0 @qiuyuzhou
